outputs.tf

```diff
@@ -14,7 +14,3 @@
  * limitations under the License.
  */
 
-output "buckets_list" {
```

Comment:
Are there really no outputs that users may be interested in? Can we output something like the forseti server? Since Terraform relies on the interpolation of variables to infer dependencies, the lack of outputs means that users can't chain Terraform resources after the forseti module.